### PR TITLE
LF: Add interface support to the Preprocessor

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
This is part of #10810

We handle only the case of Exercise Command as Fetch Command is 
currenlty used only by Canton.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala [hunk 1]
```diff
@@ -119,6 +127,8 @@ private[lf] final class CommandPreprocessor(
           choiceArgument,
         )
       case command.FetchCommand(templateId, coid) =>
+        // TODO https://github.com/digital-asset/daml/issues/10810
```
Comment (cocreature) [inline daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala:130]: This only matters for Canton right?

File: daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/PackageInterfaceSpec.scala [hunk 2]
```diff
@@ -22,6 +22,9 @@ class PackageInterfaceSpec
   import com.daml.lf.testing.parser.Implicits._
   import com.daml.lf.transaction.test.TransactionBuilder.Implicits.{defaultPackageId => _, _}
 
+  // TODO https://github.com/digital-asset/daml/issues/10810
```
Comment (cocreature) [inline daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/PackageInterfaceSpec.scala:25]: yolo

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/CommandPreprocessor.scala:130]: Yes.

**Pull request merged**